src/Fl_Text_Display.cxx

```diff
@@ -149,7 +149,7 @@ Fl_Text_Display::Fl_Text_Display(int X, int Y, int W, int H, const char* l)
   mStyleTable = 0;
   mNStyles = 0;
   mNVisibleLines = 1;
-  mLineStarts = new int[mNVisibleLines];
+  mLineStarts = new int[mNVisibleLines+1];
```

Comment:
The `for` loop is never executed because `i<mNVisibleLines` fails for `i=1` anyway. I am pretty sure this is a leftover when `mNVisibleLines` was initialised to more than 1. `new int[mNVisibleLines+1]` is not necessary.

Reply:
Result of early morning oversight. Drop the loop?